experimental/packages/opentelemetry-instrumentation-http/test/utils/assertSpan.ts

```diff
@@ -65,7 +66,19 @@ export const assertSpan = (
   );
 
   assert.strictEqual(span.links.length, 0);
-  assert.strictEqual(span.events.length, 0);
+
+  if (validations.error) {
+    assert.strictEqual(span.events.length, 1);
+    assert.strictEqual(span.events[0].name, 'exception');
+
+    const eventAttributes = span.events[0].attributes as Object;
+    assert.deepEqual(
```

Comment:
Strict version of `assert.deepEqual` is preferred: 

```suggestion
    assert.deepStrictEqual(
```